datamodel/include/data_types/system/InputSystemTypes.hpp

```diff
@@ -3,9 +3,11 @@
 
 #include <boost/bimap.hpp>
 #include <boost/assign.hpp>
+#include <boost/log/trivial.hpp>
 
 #include <X11/X.h>
 #include <X11/Xlib.h>
+#include <X11/keysym.h>
```

Comment:
This might be a bad idea, I've tried it from the start, but one mistake in the number and I got a hard time figuring out, why it wouldn't work. So I think no need to remove this

Reply:
Ok then, leave it be